docs/events.md

```diff
@@ -74,12 +74,61 @@ outageEvent.data()
 
 to fetch the whole data object, which will be an Immutable Map.
 
-### Query
+---
 
-* `toJSON()` - Returns a JSON representation of the Event
-* `toString()` - Returns a string representation of the Event, useful for serialization
-* `timestampAsUTCString()` - Returns the timestamp of the Event in UTC time.
-* `timestampAsLocalString()` - Returns the timestamp of the Event in Local time.
-* `timestamp()` - Returns the timestamp of the Event as a Date.
-* `data()` - Returns the internal data of the event, as an Immutable.Map.
-* `get(key)` - Returns the value for a specific key within the Event data. If no key is specified then 'value' is used for the key.
+### Query API
+
+#### toJSON()
+
+Returns a JSON representation of the Event
+
+#### toString()
+
+Returns a string representation of the Event, useful for serialization
+
+#### timestampAsUTCString()
+
+Returns the timestamp of the Event in UTC time.
+
+#### timestampAsLocalString()
+
+Returns the timestamp of the Event in Local time.
+
+#### timestamp()
+
+Returns the timestamp of the Event as a Date.
+
+#### data()
+
+Returns the internal data of the event, as an Immutable.Map.
+
+#### get(key)
+
+Returns the value for a specific key within the Event data. If no key is specified then 'value' is used for the key.
+
+---
+
+### Mutation API
+
+#### Event.merge([event1, event2, ...]) [Static]
+
+Creates a new Event from an array of other events. This only works under the following conditions:
+
+ * All Events are of the same time (i.e. all Events, all TimeRangeEvents or all IndexedEvents).
+ * The fields within the data of each event need to be orthogonal from each other, in that event1 might have an "a" and "b" field, then event2 should not have a "a" field but might instead have a "c" field.
```

Comment:
then event2 should not have an "a" or "b" field but might instead have a "c" field.